### PR TITLE
add blocks cache

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/2034

### Changelog

- add blocks cache

### CODE REVIEW

File: apps/explorer/lib/explorer/chain/blocks_cache.ex [hunk 1]
```diff
@@ -0,0 +1,85 @@
+defmodule Explorer.Chain.BlocksCache do
+  @moduledoc """
+  Caches the last imported blocks
+  """
+
+  alias Explorer.Repo
+
+  @block_numbers_key "block_numbers"
+  @cache_name :blocks
+  @number_of_elements 60
```
[the remark this answers is not in the document]
Comment (ayrat555) [reply apps/explorer/lib/explorer/chain/blocks_cache.ex]: @vbaranov the number of elements returned by the cache is limited to paging options page size.
https://github.com/poanetwork/blockscout/pull/2075/files#diff-138876bcd167a22ec27a92debaf1e76cR1159

[the remark this answers is not in the document]
Comment (ayrat555) [reply apps/explorer/lib/explorer/chain/blocks_cache.ex]: did you start the app locally? indexer and web app should be started on the same server  to update cache

[the remark this answers is not in the document]
Comment (vbaranov) [reply apps/explorer/lib/explorer/chain/blocks_cache.ex]: No, this PR is on stg server. Please check it.

[the remark this answers is not in the document]
Comment (ayrat555) [reply apps/explorer/lib/explorer/chain/blocks_cache.ex]: I believe the issue that you experienced was caused by the fact that the block numbers didn't have a lower limit.

I added it in https://github.com/poanetwork/blockscout/pull/2075/commits/f5ecfc9df5bfba18cb067b3f1ceffafe7e67794f

Can you please take a look again?

**Pull request merged**